randovania/data/json_data/prime3.txt

```diff
@@ -1214,25 +1386,36 @@ Asset id: 6613311822671481119
   > Pickup (Missile Expansion)
       Trivial
 
+> Event - Leviathan Shield; Heals? False
+  * Event Jungle Leviathan Shield Generator
+
 ----------------
 Hidden Court
 Asset id: 14006603351279167667
 > Dock to Hidden Court Hall; Heals? False
```

Comment:
Add connection to Top of Halfpipe with Requirements: After Golem and Screw Attack